src/vma/dev/cq_mgr.cpp

```diff
@@ -873,6 +878,9 @@ int cq_mgr::drain_and_proccess(bool b_recycle_buffers /*=false*/)
 					}
 				}
 			}
+			if (p_recycle_buffers_last_wr_id) {
+				*p_recycle_buffers_last_wr_id = (uintptr_t)wce[i].wr_id;
```

Comment:
@rosenbaumalex Where do you use the updated value in *p_recycle_buffers_last_wr_id?